### PR TITLE
Add LiteTouch Custom Component

### PR DESCRIPTION
**Pull request by patmann03**
LiteTouch Custom Component - link manifest to wheels

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @patmann03, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#39).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by patmann03** [comment]
Not sure why this is failing... I did not have a code owner initially, but added my ID.  Will this automatically re-run?  I'm a bit new to this and this is my first pull request..

File: components/litetouch.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "LiteTouch",
+    "owner": ["patmann03"],
+    "manifest": "https://github.com/patmann03/custom_components/blob/master/custom_components/litetouch/manifest.json",
```
Comment (ludeeus) [inline components/litetouch.json:4]: You need to point this to the raw url for the manifest.

```suggestion
    "manifest": "https://raw.githubusercontent.com/patmann03/custom_components/master/custom_components/litetouch/manifest.json",
```

**Pull request merged**